tests/generator/test_errors.py

```diff
@@ -999,3 +999,268 @@ def test_nm_only_supports_unicast_routes(self):
       routing-policy:
         - to: 1.2.3.4/24''', expect_fail=True)
         self.assertIn("NetworkManager only supports unicast routes", err)
+
+    def test_ignore_errors(self):
+        ''' Test if a bad netdef (eth1 in this case) will be ignored '''
+        out = self.generate('''network:
+  version: 2
+  ethernets:
+    eth0:
+      dhcp4: true
+    eth1:
+      dhcp4: yesplease
+    eth2:
+      renderer: NetworkManager
+      dhcp4: false''', expect_fail=False, skip_generated_yaml_validation=True, ignore_errors=True)
+
+        self.assertTrue(self.file_exists('10-netplan-eth0.network'))
+        self.assertTrue(self.file_exists('10-netplan-eth1.network'))
```

Comment:
nitpick (non-blocking): Should we check that the `eth1.network` config does not contain any DHCP config here?

Reply:
I think it's not necessary. There is a test in `test_netplan_parser.c:test_parser_flags_ignore_errors` that checks if `dhcp4` is still false after a failure. It should be enough to check if the netdef field has the expected value I guess, instead of checking if the configuration for it was generated or not...